examples/Billing/GetInvoices.php

```diff
@@ -0,0 +1,212 @@
+                $invoice->getServiceDateRange()->getStartDate(),
+                $invoice->getServiceDateRange()->getEndDate(),
+                self::microsToPlain($invoice->getAdjustmentsSubtotalAmountMicros()),
+                self::microsToPlain($invoice->getAdjustmentsTaxAmountMicros()),
+                self::microsToPlain($invoice->getAdjustmentsTotalAmountMicros()),
+                self::microsToPlain($invoice->getRegulatoryCostsSubtotalAmountMicros()),
+                self::microsToPlain($invoice->getRegulatoryCostsTaxAmountMicros()),
+                self::microsToPlain($invoice->getRegulatoryCostsTotalAmountMicros()),
+                $invoice->getReplacedInvoices()
+                    ? implode(
+                        ', ',
+                        iterator_to_array($invoice->getReplacedInvoices()->getIterator())
+                    ) : 'none',
+                self::microsToPlain($invoice->getSubtotalAmountMicros()),
+                self::microsToPlain($invoice->getTaxAmountMicros()),
+                self::microsToPlain($invoice->getTotalAmountMicros()),
+                $invoice->getCorrectedInvoice() ?: 'none',
+                $invoice->getPdfUrl()
+            );
+            foreach ($invoice->getAccountBudgetSummaries() as $accountBudgetSummary) {
+                /** @var AccountBudgetSummary $accountBudgetSummary */
+                printf(
+                    "  - Account budget '%s':" . PHP_EOL .
+                    "      Name (also known as Account Budget): %s" . PHP_EOL .
+                    "      Customer (also known as Account ID): %s" . PHP_EOL .
+                    "      Customer descriptive name (also known as Account): %s" . PHP_EOL .
+                    "      Purchase order number (also known as Purchase Order): %s" . PHP_EOL .
+                    "      Billing activity date range (inclusive): from %s to %s" . PHP_EOL .
+                    "      Amounts: subtotal '%.2f', tax '%.2f', total '%.2f'" . PHP_EOL,
+                    $accountBudgetSummary->getAccountBudget(),
+                    $accountBudgetSummary->getAccountBudgetName() ?: 'none',
+                    $accountBudgetSummary->getCustomer(),
+                    $accountBudgetSummary->getCustomerDescriptiveName() ?: 'none',
+                    $accountBudgetSummary->getPurchaseOrderNumber() ?: 'none',
+                    $accountBudgetSummary->getBillableActivityDateRange()->getStartDate(),
+                    $accountBudgetSummary->getBillableActivityDateRange()->getEndDate(),
+                    self::microsToPlain($accountBudgetSummary->getSubtotalAmountMicros()),
+                    self::microsToPlain($accountBudgetSummary->getTaxAmountMicros()),
+                    self::microsToPlain($accountBudgetSummary->getTotalAmountMicros())
+                );
+            }
+        }
+        // [END GetInvoices_1]
+    }
+
+    /**
+     * Converts an amount from the micro unit to the plain unit.
```

Comment:
Nit: I don't quite get the word "plain" at first.
Do we have a bit verbose explanation? Like "the unit of your currency set in your account preferences"?

Reply:
The technical term seems to be "base" instead fo "plain", does that help if I change?

Reply:
Yep. That helps.